### DISCUSSION TITLE
레벨5 이후 어떻게 할까요??

### DISCUSSION THREAD

**Discussion by zangsu** [woowacourse-teams/2024-code-zap#934] [이야기해봐요]
.

**Comment by zangsu** [comment]
- 온라인으로 진행을 할 것 같다. 
    - 몇몇 본가로 돌아가는 팀원들이 더 많아질 것이기에

## 회의록

- 짱수
    - 여러 수업, 근로가 끝나니 더 시간을 많이 쓸 수 있을 것 같다.
- 헤인
    - 사용자가 없는 상황에서 얼마나 더 서비스를 끌고 갈 수 있을지 걱정이 된다.
- 짱수 
    - 최종 코딩테스트까지만 간단하게 정해봐도 좋을 것 같다.
- 켬미 
    - 영어 면접 때문에 시간을 많이 뺏겼는데, 그 문제가 사라지니 시간을 더 투자할 수 있을 것 같다.
    - 조금 기력을 잃은 것 같긴 한데, 할 게 생기면 다시 괜찮아 질 것 같기도 하다. 
    - 할 걸 하더라도 개인 작업 위주로 하고 싶다.
- 제우스
    - 프로젝트가 오래 갈 수록 소통이 어려워 질 것이라 생각했고, 남은 사람들끼리 열심히 해야지..
    - 그치만 하고 싶은 작업은 많다. 
- 초롱
    - 열심히 하고 싶지만, 면접 일정이 계속 되니 시간이 많이 뺏기더라. 
    - 계속 열심히 하고 싶다.
- 몰리
    - 계속 하고 싶다. 해야할 것도 많고.
    - 멘탈 관리가 어려워서 신경을 많이 쓰지 못했다만 다시 열심히 하자.

**이번 스프린트는 최종 코딩테스트까지로 잡아 보는것도 좋을 것 같다.**

**Comment by zeus6768** [comment]
## 결정사항

- 최종 코테 전까지 유용한 데이터를 넣어놓자
- 홍보 내용에 익스텐션 내용을 포함하자
- 12월 9일에 홍보

**Comment by zangsu** [comment]
## 이번 스프린트 관련 이야기

- 몰리
    - 데이터를 좀 더 넣어놓아야 할 것 같다.
- 헤인
    - 생각보다 인터뷰를 해 보니 사람들이 데이터를 많이 보지 않으니 걱정하지 않아도 될 듯
- 몰리
    - 우리가 사람들을 모을 수 있게 하려면 데이터를 많이 넣어두는 것이 중요할 것 같다.
- 초롱
     - 그럼 우리가 프리코스용 코드를 많이 올려야 할 것 같다. 
- 헤인 
    - 최종 결과가 12/9에 나오니 그 전에 데이터를 많이 넣어두면 좋을듯
- 짱수
    - 익스텐션을 홍보해 보면 어떨까
- 초롱
    - 하지만 익스텐션이 프리코스 사람들에게 얼마나 매력적일지 모르겠다.
- 헤인
    - 그럼 홍보할 때 익스텐션을 추가로 홍보하는 느낌으로 ㄱㄱ
 - 초롱
     - 이벤트를 하는 것이 효과가 적은 것 같다. 오히려 광고 느낌도 난다. 
- 헤인
    - 즐겨찾기, pinned 기능을 고려해보고 싶은데 가능할까?
    - 태그가 카테고리 변경에서 유지가 되어 불편하다. 개선할 수 있을까?
    - 하지만 일정이 촉박함을 알기에 조심스럽다.
